parser.y

```diff
@@ -3035,7 +3043,8 @@ UnReservedKeyword:
 | "DYNAMIC"| "END" | "ENGINE" | "ENGINES" | "ENUM" | "ERRORS" | "ESCAPE" | "EXECUTE" | "FIELDS" | "FIRST" | "FIXED" | "FLUSH" | "FOLLOWING" | "FORMAT" | "FULL" |"GLOBAL"
 | "HASH" | "HOUR" | "LESS" | "LOCAL" | "LAST" | "NAMES" | "OFFSET" | "PASSWORD" %prec lowerThanEq | "PREPARE" | "QUICK" | "REDUNDANT"
 | "ROLE" |"ROLLBACK" | "SESSION" | "SIGNED" | "SNAPSHOT" | "START" | "STATUS" | "SUBPARTITIONS" | "SUBPARTITION" | "TABLES" | "TABLESPACE" | "TEXT" | "THAN" | "TIME" %prec lowerThanStringLitToken
-| "TIMESTAMP" %prec lowerThanStringLitToken | "TRACE" | "TRANSACTION" | "TRUNCATE" | "UNBOUNDED" | "UNKNOWN" | "VALUE" | "WARNINGS" | "YEAR" | "MODE"  | "WEEK"  | "ANY" | "SOME" | "USER" | "IDENTIFIED"
+| "TIMESTAMP" %prec lowerThanStringLitToken | "TOKUDB_DEFAULT" | "TOKUDB_FAST" | "TOKUDB_LZMA" | "TOKUDB_QUICKLZ" | "TOKUDB_SNAPPY" | "TOKUDB_SMALL" | "TOKUDB_UNCOMPRESSED" | "TOKUDB_ZLIB"
```

Comment:
I believe those are not MySQL keywords:

```
TOKUDB_DEFAULT
TOKUDB_FAST
TOKUDB_LZMA
TOKUDB_QUICKLZ
TOKUDB_SNAPPY
TOKUDB_SMALL
TOKUDB_UNCOMPRESSED
TOKUDB_ZLIB
```

https://dev.mysql.com/doc/refman/8.0/en/keywords.html#keywords-in-current-series
Would you like to move them to the `NotKeywordToken` section if so? @sunxiaoguang

Reply:
done